src/js/view/clickhandler.js

```diff
@@ -31,10 +31,6 @@ define([
                 return;
             }
 
-            if (!hasControls) {
```

Comment:
I agree with your decision to handle controls outside of this util. Do you think we should also remove the other reference to hasControls in this file as well?